test/k8sT/Policies.go

```diff
@@ -827,78 +735,158 @@ EOF`, k, v)
 		})
 	})
 
-	Context("KubernetesNetworkPolicy between server and client", func() {
+	Context("Namespaces policies", func() {
 
 		var (
-			policy = "allow-ns-b-via-namespace-selector"
+			err               error
+			namespace         = "second"
```

Comment:
This variable still has not been renamed despite my prior request to do so. Please address my prior comment.